lib/active_model/serializer/array_serializer.rb

```diff
@@ -1,13 +1,21 @@
+require 'active_model/serializer/key_format'
+require 'active_model/serializer/configuration'
+
 module ActiveModel
   class Serializer
     class ArraySerializer
       NoSerializerError = Class.new(StandardError)
+
       include Enumerable
+      include Serializer::Configuration
+      include Serializer::KeyFormat
+
       delegate :each, to: :@objects
 
-      attr_reader :root, :meta, :meta_key
+      attr_reader :root, :meta, :meta_key, :options
```

Comment:
It doesn't appear the you're using the options reader in any public contexts.  Perhaps we can make it private?  (It's just so the KeyFormat mixin can do `options[:key_format]`, right?)